src/lib/PlaylistCards.svelte

```diff
@@ -11,19 +11,44 @@
 
   export let playlists: Playlist[] = []
 
+  const rawCurationOverrides = import.meta.glob(
+    '/src/collections/playlist-curation-date-overwrites/*.md',
+    { eager: true },
+  )
+
+  let curationOverrideMap: Record<string, string> = {}
+  for (const path in rawCurationOverrides) {
+    const file = rawCurationOverrides[path]
+    const { id, curationOverwrite } = file.metadata || file
+    if (id && curationOverwrite) {
+      curationOverrideMap[id] = curationOverwrite
+    }
+  }
+
+  const beatSaverClient = beatSaverClientFactory.create()
+
   onMount(async () => {
     await getPlaylists()
+    applyCurationOverrides()
   })
 
-  const beatSaverClient = beatSaverClientFactory.create()
-
   async function getPlaylists() {
     if (playlists.length > 0) return
     let response = await beatSaverClient.fetch(
       `/playlists/latest?sort=CURATED&pageSize=${maxCards ?? 4}`,
     )
     playlists = await response.json().then((json) => json['docs'] as Playlist[])
   }
+
+  function applyCurationOverrides() {
+    playlists = playlists.map((playlist) => {
+      if (curationOverrideMap[playlist.playlistId]) {
+        return { ...playlist, curatedAt: curationOverrideMap[playlist.playlistId] }
```

Comment:
If there is more to potentially overwrite in the future, such as the cover image or title it could be easily achieved by doing the following:

1. Put the whole file content (apart from the id) in your `curationOverrideMap` (should probably then be renamed)
2. Spread the resulting object here

This would give you a little more functionality at almost no cost